bindata/network/openshift-sdn/003-rbac-controller.yaml

```diff
@@ -60,6 +60,19 @@ subjects:
   name: sdn-controller
   namespace: openshift-sdn
 
+- apiGroups:
```

Comment:
Mis-paste - this needs to be in a ClusterRole, not a ClusterRoleBinding.

Reply:
Oops. It is.